bucket/ghostscript.json

```diff
@@ -1,19 +1,19 @@
 {
     "homepage": "https://www.ghostscript.com",
-    "version": "9.26",
+    "version": "9.27",
     "license": "AGPL-3.0",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs926/gs926aw64.exe#/dl.7z",
-            "hash": "sha512:4c961b852bd207e9a1cae00355c5e582ad047dec54912ae5e7d326c4eacc0c8300bf49164d7ba5817f9ab5a7b9cb5d60f235b162dd30c0d6e8706566f7e0a605",
+            "url": "https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs927/gs927w64.exe#/dl.7z",
+            "hash": "sha512:49fc973bfae12f366a837bcc39d93e3d8e3c88359f7d45279df899f0b2941c264efc47815966867d332b32e87ca8d88697edf7792b162645ed7a549cecc373b2",
             "bin": [
                 "bin\\gswin64.exe",
```

Comment:
If you want to keep binaries, i would really suggest to add new one, renamed without architecture.

`gswin64c -> gswin` I assume, that `c` means console

Reply:
For binary names, some apps that use ghostscript are looking for such file names, so renaming is not prefered.

Reply:
> to add new one

I did not said delete current ones. I said add new one with renames, so user can freely type `gscript` | `gs` instead of `gswin64c`
When you add renames, it will not break anything

Reply:
Got it.